spk/radarr/Makefile

```diff
@@ -1,22 +1,21 @@
 SPK_NAME = radarr
 SPK_VERS = $(shell date +%Y%m%d)
-SPK_REV = 14
+SPK_REV = 15
 SPK_ICON = src/radarr.png
 
 REQUIRED_DSM = 5.0
```

Comment:
Should this be increased?

Reply:
I don't think it's necessary. I can't test on DSM 5.0, but we release mainly on DSM 6.x anyway. Upping it would just block anyone from building it on lower DSM versions locally. IMHO it could just be removed.